app/assets/stylesheets/home.scss

```diff
@@ -21,11 +20,25 @@
   }
 }
 
-div.module.text-center {
-  border-radius: 50px;
+.pie-size {
+  position: relative;
+  height: 300px;
+  background: asset-url("energy.jpg") no-repeat center center;
 }
 
-.learn {
-  background-color: #d5ffea;
-  border-radius: 50px;
+.vertical-center {
+  position: absolute;
+  top: 25%;
+
+  input {
+    border-style: none;
```

Comment:
Properties should be ordered background-color, border-style, color, font-size, padding